src/applications/mhv/secure-messaging/components/ComposeForm/ComposeForm.jsx

```diff
@@ -340,11 +354,20 @@ const ComposeForm = props => {
         <VaModal
           modalTitle={saveError.title}
           onPrimaryButtonClick={() => setSaveError(null)}
+          onSecondaryButtonClick={() => {
+            setSaveError(null);
+          }}
           onCloseEvent={() => {
             setSaveError(null);
             focusElement(lastFocusableElement);
           }}
           primaryButtonText="Continue editing"
+          secondaryButtonText={
+            attachments
+              ? ErrorMessages.ComposeForm.UNABLE_TO_SAVE_DRAFT_ATTACHMENT
+                  .saveDraft
+              : null
```

Comment:
We can probably shorten this up to just use the constant, ErrorMessages.ComposeForm..............